manifest_template.yml

```diff
@@ -97,21 +109,26 @@ apigee:
         environments: [ {{ ENV.name }} ]
         proxies:
           - e-referrals-service-api-{{ VARIANT.service | default(VARIANT.name) }}-{{ ENV.name }}
+          { % if ENV.name == 'dev' % }
+          - identity-service-dep-{{ ENV.name }}
+          - identity-service-mock-{{ ENV.name }}
+          {% else %}
```

Comment:
It might also be slightly cleaner if this block was written as an `if` also rather than an `else` block. It would also ensure that if we add another environment the proxies within the `else` block aren't inadvertently added. I think you'd end up with something like
```
{ % if ENV.name == 'dev' % }
...
{ % endif % }
{ % if ENV.name == 'internal-dev' % }
...
{ % endif % }